setup.py

```diff
@@ -8,8 +8,8 @@
     include_package_data=True,
     install_requires=[
         'cached_property',
-        'django>=1.8,<1.12',
-        'django-mptt',
+        'django==1.11',
```

Comment:
I suspect you didn't intend to pin to exactly 1.11 (which excludes 1.11.1, 1.11.2, etc.). As a best effort, we should also _try_ to support 1.8 through 1.10 until we hit an incompatibility. I think that means we can revert this line?